lib/resources/example_json/r4_examples_location.rb

```diff
@@ -95,5 +95,244 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_LOCATION_ADDRESS_ENTRY ||= {
+      "resourceType": 'Location',
+      "id": '982243103',
+      "meta": {
+        "versionId": '0',
+        "lastUpdated": '2019-03-29T14:40:50Z'
+      },
+      "text": {
+        "status": 'generated',
+        "div": '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Location</b></p><p><b>Name</b>: '\
+        'Anushka Org One</p><p><b>Alias</b>: ADOne</p><p><b>Managing Organization</b>: Anushka Org One</p>'\
+        '<p><b>Physical Type</b>: Facility(s)</p><p><b>Status</b>: Active</p></div>'
+      },
+      "status": 'active',
+      "name": 'Anushka Org One',
+      "alias": [
+        'ADOne'
+      ],
+      "mode": 'instance',
+      "address": {
+        "use": 'work',
+        "text": "111th West st\nMalvern, PA 19355\nUSA",
+        "line": [
+          '111th West st'
+        ],
+        "city": 'Malvern',
+        "state": 'PA',
+        "postalCode": '19355',
+        "country": 'USA'
+      },
+      "physicalType": {
+        "coding": [
+          {
+            "system": 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/222',
```

Comment:
change domain to ec2458f2-1e24-41c8-b71b-0e701af7583d

Reply:
addressed https://github.com/cerner/fhir.cerner.com/pull/749/commits/4e79521abc896ef5a9ef28e79b25a6842781919b